src/basic/cgroup-util.h

```diff
@@ -183,9 +183,13 @@ typedef enum CGroupUnified {
 int cg_path_open(const char *controller, const char *path);
 int cg_cgroupid_open(int fsfd, uint64_t id);
 
+typedef enum CGroupPidFlags {
```

Comment:
BTW, maybe better to merge this enum into CGroupFlags in the above? Then, we can set the new flag to `cg_migrate()` if necessary.

Reply:
I'm not sure. I don't understand cg_migrate well enough, but to me its code looks like it can't work properly if the pid returned by cg_read_pid is ever zero.

So to me, the only two options for this function seem to be to either ignore all zeros and skip over them, or don't ignore them and return -EIO when it encounters one, which would keep its behaviour identical to how it is right now.